core/kazoo_number_manager/src/knm_telnyx_util.erl

```diff
@@ -66,13 +62,10 @@ req(Method, Path) ->
 
 -spec req(atom(), [nonempty_string()], kz_json:object()) -> kz_json:object().
 req('post', ["number_searches"], JObj) ->
-    case kz_json:get_value([<<"search_descriptor">>, <<"prefix">>], JObj) of
-        <<"800">> -> rep_fixture("telnyx_tollfree_search.json");
-        'undefined' ->
-            case kz_json:get_value([<<"search_descriptor">>, <<"country_iso">>], JObj) of
-                <<"GB">> -> rep_fixture("telnyx_international_search.json");
-                'undefined' -> rep_fixture("telnyx_npa_search.json")
-            end
+    case kz_json:get_integer_value([<<"search_type">>], JObj) of
+        1 -> rep_fixture("telnyx_npa_search.json");
```

Comment:
Generally dislike `magic` numbers like these. Can you `-define(NICE_NAME, 1)` and use `?NICE_NAME` instead? I have no idea what 1, 2, and 3 mean in this context...

Reply:
Yeah definitely a cleaner way of doing it. Will make the change